src/go/rpk/pkg/cli/debug/bundle/bundle_linux.go

```diff
@@ -578,6 +579,37 @@ func saveCmdLine(ps *stepParams) step {
 	}
 }
 
+func saveRedpandaProc(ps *stepParams) step {
+	return func() error {
+		pid := getRedpandaPID(ps.fs)
+		if pid == 0 {
+			return errors.New("Unable to locate Redpanda PID")
+		}
+		targets := []string{"cgroup", "cmdline", "limits", "sched", "stack", "status"}
```

Comment:
Is `stack` necessary? it requires root permissions.

Reply:
Hmm, probably not - I'm open to dropping it if you think that's a problem. We do have other things we collect that require root though.

Reply:
If it's unnecessary, I'll say it's better to drop it, as the whole `proc` collection will fail for this error. And tbf, we shouldn't have the other commands that require sudo. We should only care for those accessible to the redpanda user.

Reply:
Hmm, it shouldn't stop the others as we run them in `multierror.Group`. Everything will succeed or fail on its own, and we'll just log the ones that don't work. That said, I've got no personal attachment to keeping it, I'll remove it :)